contrib/win32/win32compat/misc.c

```diff
@@ -1466,3 +1466,77 @@ is_absolute_path(char *path)
 	
 	return retVal;
 }
+
+/* return 0 - in case of failure */
+int
+create_directory_withsddl(char *path, char *sddl)
```

Comment:
if retuning int, follow POSIX convension. return -1 on failure, 0 on success and set error in errno.

Reply:
I followed the win32 API convention.. 

https://msdn.microsoft.com/en-us/library/windows/desktop/aa363855(v=vs.85).aspx

CreateDirectory() returns non-zero incase of success, 0 incase of failure